### PR TITLE
Commenting. Implement disable/enable meeting commenting use cases.

### PR DESCRIPTION
**Pull request by AndreiGanichev**
Implement disable/enable meeting commenting use cases.

### CODE REVIEW

File: src/Modules/Meetings/Domain/MeetingCommentingConfigurations/MeetingCommentingConfiguration.cs [hunk 1]
```diff
@@ -29,5 +32,27 @@ private MeetingCommentingConfiguration()
 
         internal static MeetingCommentingConfiguration Create(MeetingId meetingId)
             => new MeetingCommentingConfiguration(meetingId);
+
+        public void EnableCommenting(MemberId enablingMemberId, MeetingGroup meetingGroup)
```
Comment (kgrzybek) [inline src/Modules/Meetings/Domain/MeetingCommentingConfigurations/MeetingCommentingConfiguration.cs:36]: Please fix the warning *'public' members should come before 'internal' members*

File: src/API/CompanyName.MyMeetings.API/Modules/Meetings/MeetingCommentingConfiguration/MeetingCommentingConfigurationController.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        public async Task<IActionResult> DisableCommenting(Guid meetingId)
+        {
+            await _meetingsModule.ExecuteCommandAsync(new DisableMeetingCommentingConfigurationCommand(meetingId));
+            return Ok();
+        }
+
+        [HttpPatchAttribute("enable")]
+        [HasPermission(MeetingsPermissions.EditMeetingCommentingConfiguration)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<IActionResult> EnableCommenting(Guid meetingId)
```
Comment (kgrzybek) [inline src/API/CompanyName.MyMeetings.API/Modules/Meetings/MeetingCommentingConfiguration/MeetingCommentingConfigurationController.cs:35]: In my opinion, we should have permissions per action. In this case DisableMeetingCommenting and EnableMeetingCommenting. What do you think?

**Review by kgrzybek — CHANGES_REQUESTED** [review]

File: src/API/CompanyName.MyMeetings.API/Modules/Meetings/MeetingCommentingConfiguration/MeetingCommentingConfigurationController.cs (hunk 2 above)
Comment (AndreiGanichev) [reply src/API/CompanyName.MyMeetings.API/Modules/Meetings/MeetingCommentingConfiguration/MeetingCommentingConfigurationController.cs]: I think you are right. Permission per action gives us maximum flexibility to configure authorization without changing code of controllers. We can do it by managing user's roles and role's permissions. And moreover such granular permissions are more explicit and allows us not to think hard about meaningful permission name :)

File: src/Modules/Meetings/Domain/MeetingCommentingConfigurations/MeetingCommentingConfiguration.cs (hunk 1 above)
Comment (AndreiGanichev) [reply src/Modules/Meetings/Domain/MeetingCommentingConfigurations/MeetingCommentingConfiguration.cs:33]: sure! i thought i fixed them all.

**Comment by AndreiGanichev** [comment]
I have a problem with SIT in Payments module. Different tests failed in checks after two different pushes. Can a probe not get a value after given timeout?

**Comment by kgrzybek** [comment]
I did not have an issue on my local machine but there is some issue on CI I see. I will investigate that. Meanwhile, I accept this PR.

**Pull request merged**